src/Notification/Notification.less

```diff
@@ -1,4 +1,11 @@
 /* Popup.css */
+.moon {
+	.moon-notification {
+		padding: 27px 39px 27px 39px;
+		display: table;
+	}
+}
+
```

Comment:
Hmm, I saw padding in .moon-notification is overridden by padding in .moon's.

http://10.186.120.46/~yeram.choi/dist/